glean-core/src/util.rs

```diff
@@ -142,7 +146,7 @@ pub mod floating_point_context {
     impl FloatingPointContext {
         pub fn new() -> Self {
             let mut current: size_t = 0;
-            let _err = unsafe { _controlfp_s(&mut current, PC_64, MCW_PC) };
+            let _err = unsafe { _controlfp_s(&mut current, PC_64 | RC_CHOP, MCW_PC | MCW_RC) };
```

Comment:
Oh, interesting. Is python changing this flag internally as well? Can you update the comment at the top of this class mentioning that we set rounding in addition to the precision?

Reply:
Yes -- the rounding was changing on 64-bit, the precision was changing on 32-bit.  I'll update the comment.